src/frontends/jax/src/op/reduce_window_max.cpp

```diff
@@ -0,0 +1,95 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <cstddef>
+#include <cstdint>
+#include <vector>
+
+#include "openvino/core/node.hpp"
+#include "openvino/core/node_output.hpp"
+#include "openvino/frontend/jax/node_context.hpp"
+#include "openvino/op/constant.hpp"
+#include "openvino/op/max_pool.hpp"
+#include "openvino/op/transpose.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace jax {
+namespace op {
+
+using namespace ov::op;
+
+OutputVector translate_reduce_window_max(const NodeContext& context) {
```

Comment:
`reduce_window_sum` and `reduce_window_max` are very similar, could you unify them as `reduce_window_common` function?